### ISSUE TITLE
Initialization of weights

### ISSUE THREAD

**Issue by shshojaei** [cyun-404/PieESRGAN#3]
Hi :) what is the Initialization of weights? I downloaded the weights but These are not initial values, because when I use them, I get the following error: 
 `Expected values to be greater or equal to 0, got -0.03592266887426376`
for this line:
```
# Update the weights of the generator model.
pieapp_loss: torch.Tensor = piq.PieAPP(reduction='none', stride=128)(hr.detach(), sr)
```

**Comment by cyun-404** [comment]
Hello :) I want to see more of your errors. Can you capture the entire error statement so that I can see where the error is coming from?
Maybe it's because the value is less than zero when Torch processes loss, but I think we need to check the entire error message to check more details.

**Comment by shshojaei** [comment]
![error](https://user-images.githubusercontent.com/79797720/199312673-f9eb9337-47f8-4655-b6b9-c4ed0822a338.png)

**Comment by shshojaei** [comment]
in train.py file

**Comment by cyun-404** [comment]
How about remove the 97~101 line? I think it's just range problem, but in Pieapp loss can be come out less than 0 value.
For me, I dont have that problem. It is may be differece of python version.

**Comment by shshojaei** [comment]
Which version of Python did you use?

**Comment by cyun-404** [comment]
I can't check the version of the experimental computer because I leave out the lab. However, when I saw the issue, there was the same error when I runed it to the new computer, so I deleted the if~~~ sentence as I replied before and checked that it runing.

**Comment by cyun-404** [comment]
Is the problem solved?

**Comment by shshojaei** [comment]
> Is the problem solved?

No, I didn't want to remove part of the code, thanks for your reply.
